_toc.yml

```diff
@@ -1,5 +1,59 @@
 - file: index
 - chapters:
+  - file: tutorials/start_index
+    sections:
+    - file: tutorials/fundamentals/installation
+    # - file: tutorials/absolute_beginners
+    - file: tutorials/fundamentals/getting_started
+    - file: plugins/stable/find-and-install-plugin
+    - file: tutorials/fundamentals/viewer
+    # The following will be replaced by
+    # - file: tutorials/fundamentals/layers/index
+    - file: tutorials/fundamentals/image
+    - file: tutorials/fundamentals/labels
+    - file: tutorials/fundamentals/points
+    - file: tutorials/fundamentals/shapes
+    - file: tutorials/fundamentals/surface
+    - file: tutorials/fundamentals/tracks
+    - file: tutorials/fundamentals/vectors
+    title: Getting started
+  - file: tutorials/index
+    sections:
+    - file: tutorials/segmentation/index
+    - file: tutorials/tracking/index
+    - file: tutorials/applications/annotate_points
+    - file: tutorials/applications/dask
+    # - file: tutorials/particle_trackpy
+    - file: tutorials/further-resources/napari-workshops
+    title: Tutorials
+  - file: howtos_index
+    sections:
+    # - file: tutorials/fundamentals/layers/index
+    - file: tutorials/applications/napari_imageJ
+    - file: guides/stable/perfmon
+    - file: guides/stable/docker
+    - file: guides/stable/connecting_events
```

Comment:
I don't know if the files in the how-to section are in a specific order for a reason, but I think that "connecting events" is a very important topic and should maybe be first in the list. (For similar reasons, I might also put "perfmon" last, since I'd expect only pretty expert people are likely to be the ones doing that)

Reply:
Sounds good, no particular reason so I'll just reorganize, thanks!